teiid-feature-pack/pom.xml

```diff
@@ -16,7 +16,6 @@
     <module>rest-service-feature-pack</module>
     <module>wildfly-integration-feature-pack</module>
     <module>olingo-feature-pack</module>
-    <module>wildfly-overlay-dist</module>
```

Comment:
Because, wildfly-overlay-dist depend on a series of connectors feature pack, we have to put wildfly-overlay-dist after all connectors's build.